server/aws/README.md

```
@@ -0,0 +1,164 @@
+pnpm start:verifier
+```
+
+You can also run from the monorepo root using the package filter:
+
+```bash
+pnpm -F @trustknots/server-aws start:issuer
+```
+
+### Startup confirmation
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)